MAUI/Scheduler/load-on-demand.md

```diff
@@ -0,0 +1,296 @@
+---
+layout: post
+title: Load appointments on demand in .NET MAUI Scheduler | Syncfusion
+description: Learn here all about to load appointments on demand from visible dates in Syncfusion .NET MAUI Scheduler (SfScheduler) control, its elements, and more. 
+platform: maui
+control: SfScheduler
+documentation: ug
+---
+
+# Load on demand in .NET MAUI Event Scheduler (SfScheduler).
+
+The Scheduler provides the capability to display an interactive view when the view is changed, or swipe between the views, and also to load appointments on-demand with a loading indicator, thereby improving loading performance when there are appointments range for multiple years.
+
+## Load appointments on demand
+
+It allows you to load appointments on-demand for the visible dates. The `ShowBusyIndicator` property can be used to start and stop the loading indicator animation before and after appointments are loaded.
+
+These can be achieved in two ways,
+
+* QueryAppointments event
+* QueryAppointments command
+
+### QueryAppointments event
+
+This event occurs after the view of the scheduler or the visible dates are changed and also allows `AppointmentsSource` for the visible dates to be loaded in on-demand. Below is a list of the arguments
```

Comment:
/// In the scheduler agenda view, the QueryAppointments event is used to load more appointments when the new month is loaded on view, whereas the new month appointments are added in scheduler AppointmentsSource.
        /// Other than agenda view, the scheduler AppointmentsSource can be reset for a new visible date range to improve appointment loading performance.